common/static/common/js/discussion/discussion_module_view.js

```diff
@@ -183,7 +183,7 @@ NewPostView */
                         el: self.$('article#thread_' + thread.id),
                         model: thread,
                         mode: 'inline',
-                        context: self.context,
+                        context: 'standalone',
```

Comment:
It is so frustrating that this logic is duplicated everywhere. IMO you shouldn't change the context, because that has been provided by the server. I think the test is wrong in discussion_thread_edit_view.js. If you look at the equivalent line in new_post_view.js, it is:

```javascript
if (this.isTabMode()) {
```

I suggest making the edit post use the same check.